### ISSUE TITLE
View layout without partial

### ISSUE THREAD

**Issue by Kikobeats** [balderdashy/sails#2400]
first, I didn't find difference between res.view and res.render... why have both methods? Yes, I check the documentation, but how I know that the view is in memory? wtf.

Second, I have a layout template that use partial view from the controller (and data to bind with the views). But have a case of use where I want to use the layout with data bind but without partial view. Someting as:

```
res.render undefined, {name:'kiko'}
```

but throw a error. I tried with `null`, `{}`, `''`... the only solution that I found is created a empty partial view

```
res.render 'empty',  {name:'kiko'}
```

but in my opinion it's horrible.

Is it not supported?

**Comment by CWyrtzen** [comment]
@Kikobeats what is the resolution to this?

**Comment by sailsbot** [comment]
Thanks for posting, @Kikobeats.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
